_docker/drupal/drupal-filesystem/web/sites/default/settings.php

```diff
@@ -753,6 +753,5 @@
   include __DIR__ . '/rhd.settings.php';
 }
 
-$settings['install_profile'] = 'lightning';
```

Comment:
Was this change intentional @robpblake ?

Reply:
@jordanpagewhite Yes. This is no longer required for Drupal 8.

cheers,

Rob

Reply:
Ah, cool! I wasn't aware of this change. I see the change record now in the 8.3.x branch though. Thanks!